src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -544,6 +567,11 @@ private string UFormatDateString(DateTime dateTime)
         }
 
         #endregion
+
+        private const string DateAndFormat = "DateAndFormat";
```

Comment:
It is parameter set names. So I'd add this in the consts:
```suggestion
        private const string DateAndFormatParameterSet = "DateAndFormat";
```

Reply:
I think that if the parameter set name constants have the suffix `ParameterSet`, it would be better to make them members of the static inner class, as before.

Reply:
It is our common pattern. If you see a const with the suffix in code you understand that it is a parameter set name.